image-rs/src/image.rs

```diff
@@ -146,7 +148,33 @@ impl Default for ImageClient {
         }
     }
 }
-
+/// set_image creates a mapping backed by image block device <source_device_path> and
+/// decoding <verity_options> for in-kernel verification. And mount the verity device
+/// to <mount_path> with <mount_type>.
+/// It will return the verity device path if succeeds and return an error if fails .
+pub fn mount_image_block_with_integrity(
```

Comment:
Do we need the counterpart `unmount_image_block_with_integrity()` to clean up all resources?